### PR TITLE
Dev models

### PR DESCRIPTION
**Pull request by DipandaAser**
#### This pull request add models generated by using the migrations description

**Why ?**
We need to interact with models to bind data stored in bd with logic

**How ?**
By adding struct for each table present in [backend/migrations/001_initial.sql](https://github.com/osscameroon/yotas/pull/38/files#diff-56b8a16c65214d799a928a0f5801581f4982dd6781a6dd7058d8619f4cb05d2c)

**Steps to verify:**
Verify if each table present in [backend/migrations/001_initial.sql](https://github.com/osscameroon/yotas/pull/38/files#diff-56b8a16c65214d799a928a0f5801581f4982dd6781a6dd7058d8619f4cb05d2c) are created.

** Screenshots (optional)

### CODE REVIEW

File: backend/auth/models.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import (
+	"gorm.io/gorm"
+	"time"
+)
+
+type Users struct {
+	gorm.Model
+	Name      string
+	Email     string
+	GithubId  string
```
Comment (gtindo) [inline backend/auth/models.go:12]: We should add github_token on the model

We will use that to fetch his github profile

File: backend/migrations/001_initial.sql [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    UNIQUE(email, github_id),
+    PRIMARY KEY(Id)
+);
+
+CREATE TABLE IF NOT EXISTS users (
+    id INT NOT NULL AUTO_INCREMENT,
+
+    `name` VARCHAR(150),
+    email VARCHAR(150) NOT NULL,
+    github_id VARCHAR(150) NOT NULL,
```
Comment (gtindo) [inline backend/migrations/001_initial.sql:26]: We should add the github token

**Review by gtindo — CHANGES_REQUESTED** [review]

**Comment by elhmn** [comment]
> #### This pull request add models generated by using the migrations description
> **Why ?**
> We need to interact with models to bind data stored in bd with logic
> 
> **How ?**
> 
> **Steps to verify:**
> None
> 
> ** Screenshots (optional)

Can you fill this in ?

**Review by gtindo — APPROVED** [review]
LGTM

**Pull request merged**